pkg/operator/ceph/spec/spec.go

```diff
@@ -413,3 +415,41 @@ func StoredLogAndCrashVolumeMount(varLogCephDir, varLibCephCrashDir string) []v1
 		},
 	}
 }
+
+func SetPodPlacement(pod *v1.PodSpec, p rook.Placement, requiredDuringScheduling, preferredDuringScheduling bool,
```

Comment:
I'd suggest moving this method to the `pkg/operator/k8sutil` package as long. There isn't anything ceph-specific here.

Reply:
This method is only adding node anti-affinity, right? Should the method name be more specific such as `SetNodeAntiAffinityForPod()`?